.github/scripts/jpackage.bat

```diff
@@ -30,4 +30,4 @@ REM set MODULES=java.desktop,java.logging,java.naming,java.prefs,java.security.j
 --win-menu ^
 --win-menu-group "Scene Builder" ^
 --win-per-user-install ^
---win-shortcut
\ No newline at end of file
+--win-shortcut
```

Comment:
For some reason the GH editor added new line at the end, but this should not be an issue.